Sources/Fullscreen/BetaFeatureView/BetaFeatureView.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public struct BetaFeatureViewModel {
+    public let iconImage: UIImage?
+    public let title: String
+    public let description: String
+    public let firstButtonTitle: String
+    public let secondButtonTitle: String
+
+    public init(
+        iconImage: UIImage?,
+        title: String,
+        description: String,
+        firstButtonTitle: String,
+        secondButtonTitle: String
+    ) {
+        self.iconImage = iconImage
+        self.title = title
+        self.description = description
+        self.firstButtonTitle = firstButtonTitle
+        self.secondButtonTitle = secondButtonTitle
+    }
+}
+
+public protocol BetaFeatureViewDelegate: AnyObject {
+    func betaFeatureViewDidSelectFirstButton(_ view: BetaFeatureView)
+    func betaFeatureViewDidSelectSecondButton(_ view: BetaFeatureView)
+}
+
+public class BetaFeatureView: UIView {
+    // MARK: - Public properties
+    public weak var delegate: BetaFeatureViewDelegate?
+
+    // MARK: - Private properties
+    private lazy var betaFlagView: UIImageView = {
+        let imageView = UIImageView(withAutoLayout: true)
+        imageView.image = UIImage(named: .betaPill)
+        return imageView
+    }()
+
+    private lazy var iconImageView = UIImageView(withAutoLayout: true)
+    private lazy var titleLabel: Label = {
+        let label = Label(style: .title3Strong, withAutoLayout: true)
+        label.textAlignment = .center
+        label.setContentHuggingPriority(.defaultHigh, for: .vertical)
+        return label
+    }()
+
+    private lazy var bodyLabel: Label = {
+        let label = Label(style: .body, withAutoLayout: true)
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        label.setContentHuggingPriority(.defaultHigh, for: .vertical)
+        return label
+    }()
+
+    private lazy var firstButton: Button = {
+        let button = Button(style: .callToAction, withAutoLayout: true)
+        button.addTarget(self, action: #selector(handleTapOnFirstButton), for: .touchUpInside)
+        button.setContentHuggingPriority(.defaultHigh, for: .vertical)
+        return button
+    }()
+
+    private lazy var secondButton: Button = {
+        let button = Button(style: .flat, withAutoLayout: true)
+        button.addTarget(self, action: #selector(handleTapOnSecondButton), for: .touchUpInside)
+        button.setContentHuggingPriority(.defaultHigh, for: .vertical)
+        return button
+    }()
+
+    private let betaPillSize = CGSize(width: 52, height: 24)
```

Comment:
Does it need to be an instance var? It seems like it's used only once in `setup`?

Reply:
it doens't have to, but to me it should more intention rather than putting the concrete values in the constraint as "magic" values

Reply:
Still can be a local var in `setup` I think.